src/types/index.ts

```diff
@@ -34,3 +34,10 @@ export interface BaseResponse {
 export interface BaseComponentProps {
     children?: ReactNode;
 }
+
+export interface OpenGraph {
+    'en-US': {
+        image: string;
+        title: string;
+    };
+}
```

Comment:
Eventually we should probably get this from Supabase somehow... but just hardcoding for now.